frontend/src/app/register/register.ctrl.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Controller for register page
+ */
+angular.module('flocs.user')
+.controller('registerCtrl', ['$scope','$location','$log','userDao', 
+    function($scope,$location,$log,userDao){
+
+	    function send(){
+            var username = $scope['username'];
+            $log.log($scope['username']);
+            var firstname = $scope['firstname'];
+            var lastname = $scope['lastname'];
+            var email = $scope['email'];
+			var passwd = $scope['password'];
+			if (userDao.registerUser(username,firstname,lastname,email,passwd) === true){
+				$location.url("/succes.html");
```

Comment:
do not use $location.url() - you navigate to the specific url, but you should go to the states declared in app.mdl.js (there are no hard coded urls)